share/src/test/java/org/alfresco/web/config/properties/AIMSConfigTest.java

```diff
@@ -41,9 +40,8 @@
 
 public class AIMSConfigTest extends BaseTest
 {
-
-    public static final String PROPERTIES_RESOURCES = "classpath*:alfresco/module/*/share-config.properties";
-    private static final String TEST_CONFIG_AIMS_BASIC_XML = "test-config-aims-basic.xml";
+    // public static final String PROPERTIES_RESOURCES = "classpath*:alfresco/module/*/share-config.properties";
```

Comment:
Why this class has been commented out?

Reply:
@cristina-sirbu I've commented the AIMS tests, the AIMSConfig was changed considerably. I've added I minimal set of tests for the updated AIMSConfig.